sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/collectionOperations.scala

```diff
@@ -888,6 +888,178 @@ case class MapFromEntries(child: Expression)
     copy(child = newChild)
 }
 
+@ExpressionDescription(
+  usage = """
+    _FUNC_(map[, ascendingOrder]) - Sorts the input map in ascending or descending order
+      according to the natural ordering of the map keys. The sorting algorithm used is
+      an adaptive, stable and iterative merge sort algorithm. If the input map is empty,
```

Comment:
No need to expose the implementation details like `merge-sort`. It is enough to mention user-visible behavior like `stable`.